src/store/sagas/gameFlowSaga.ts

```diff
@@ -247,6 +249,23 @@ export default function* gameFlowSaga() {
 
   // Watch for game completion
   yield takeEvery(completeGame.type, function* (): Generator<any, void, any> {
+    // NEW: Check for whitewash before proceeding to summary
+    const gameProgress = yield select(selectGameProgress);
+    const isWhitewash = gameProgress.scores.team1 === 250; // MAX_BID = 250
```

Comment:
nit: use `isWhiteWash` function from gameUtils
```suggestion
    const isWhitewash = isWhiteWash(gameProgress.scores)
```